XRTK-Core/Packages/com.xrtk.core/Services/DiagnosticsSystem/MixedRealityDiagnosticsSystem.cs

```diff
@@ -124,7 +124,7 @@ public Transform DiagnosticsRoot
                 if (diagnosticsRoot == null)
                 {
                     diagnosticsRoot = new GameObject("Diagnostics").transform;
-                    diagnosticsRoot.parent = MixedRealityToolkit.CameraSystem?.CameraRig.PlayspaceTransform;
+                    diagnosticsRoot.transform.SetParent(MixedRealityToolkit.CameraSystem?.MainCameraRig.PlayspaceTransform, false);
```

Comment:
We should check if the camera system is null here and fall back to `CameraCache.Main.parent`